antora-playbook.yml

```diff
@@ -34,24 +34,24 @@ content:
   # What branch, main or some staging, e.g. cloud-dev?
   sources:
     - url: https://github.com/tigergraph/doc-site.git
-      branches: [main]
+      branches: [cloud-dev]
     #- url: https://github.com/tigergraph/doc-site.git
     #  branches: [staging]
 
       # (3) Purpose: What is the content: remote or local?
       # What versions (Previews will usually have extra versions).
       start_paths: home
     - url: https://github.com/tigergraph/cloud-docs.git
-      branches: [main]
-      start_paths: [modules/cloud, modules/cloudbeta]
+      branches: [dev]
+      start_paths: [modules/cloud, modules/cloud4]
```

Comment:
@samzhou2 So I assume you doing a simple rename of the module cloud-docs/cloudbeta to cloud-docs/cloud4 and/or changing the name(?) property in the antora.yml file?

Reply:
Added redirects from cloudbeta to cloud4